webapp/package.json

```diff
@@ -44,7 +44,8 @@
     ],
     "moduleNameMapper": {
       "^@/(.*)$": "<rootDir>/src/$1",
-      "^~/(.*)$": "<rootDir>/$1"
+      "^~/(.*)$": "<rootDir>/$1",
+      "\\.(css|less)$": "identity-obj-proxy"
```

Comment:
What is that? :thinking:

Reply:
it is for mocking CSS modules in Jest
so we are importing the CSS for cropperjs, but it breaks the tests, so we need to mock it
https://jestjs.io/docs/en/webpack#mocking-css-modules